.circleci/config.yml

```diff
@@ -51,7 +43,7 @@ commands:
     steps:
       - run:
           name: Install Node dependencies
-          command: node tools/ci.js
+          command: npm install 
```

Comment:
I think we can delete the `tools/ci.js` file now. Doesn't seem to be used anywhere else.